<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_in</class>
 <widget class="QMainWindow" name="sign_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/green.png);</string>
    </property>
    <property name="text">
     <string>username:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/green.png);</string>
    </property>
    <property name="text">
     <string>password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="userNameEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/blue.png);</string>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/blue.png);</string>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
   </widget>
   <addaction name="menuLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>sign_in</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
